e2e/haitaton_suomifi_login.spec.ts

```diff
@@ -237,40 +279,41 @@ test('Toiminnallisia testejä', async ({ page }) => {
   // Vaihtele käyttöoikeuksia
   await page.getByTestId('save-form-btn').click();
   await page.getByRole('button', { name: 'Käyttäjähallinta' }).click();
-  await expect(page.getByTestId('kayttooikeustaso-1')).toContainText('Katseluoikeus');
+  const row = page.locator('tr:has-text("Testi Automaatio")');
+  await expect(row.filter({ hasText: 'Katseluoikeus' })).toBeVisible();
```

Comment:
For me the test fails here: `<element(s) not found>`. Not sure if it's just on my machine.

Reply:
Not sure if it works just on my machine either. I have tried to run the test multiple times now and it does not fail. For me. Maybe I can merge and we'll see what gets broken in real E2E environment in test.